Documentation/ExtensionInstallation/Index.rst

```diff
@@ -55,6 +63,86 @@ In the backend:
 8. Click on the icon with a :guilabel:`"+"` sign for your extension
    in the :guilabel:`"A/D"` column.
 
+.. _uninstall_extension_without_composer
+
+Uninstall an Extension Without Composer
+=======================================
+
+If you installed TYPO3 via composer you should uninstall Extensions via composer.
+
+Check Dependencies
+------------------
+
+First find out, which other extensions and functions of your TYPO3-Installation are dependent on the extension you
+want to deinstall. You can find out about the dependencies by checking the
+`Extension Repository <https://extensions.typo3.org/>`__. Look for the extension
+you want to deinstall and the others you have installed. Read in each extensions manual the sections 'Dependencies' and
+'Reverse dependencies'.
+
+Contemplate, whether you made referrals to the extension in any setup, config or other TypoScript-section. Check if you
+included plugins from the extension in your page. Think of the results of deleting them and finally do it.
+
+If you are working locally or on a test server you might as well try to deinstall the extension. The Extension Manager
+warns you about dependencies that are written in an extensions :file:`ext_emconf.php` constrains section. Note however
+that you depend on the extensions developers faithfully noting all dependencies in this config file.
+
+If you get an exception and can't even access the Extension Manager anymore because of it, you can deinstall / install
+extensions manually with :file:`PackageStates.php` as a last resort, see :ref:`install-extension-manually`
+
+.. tip::
+   Be sure not to uninstall extensions by trial and error on life systems, expecially not under time pressure.
+
+.. _uninstall-extension-backend:
+
+Uninstall / Deinstall / Remove Extension via TYPO3 Backend
```

Comment:
Why Uninstall **and** Deinstall? Should be "Install / Uninstall /  Remove"?

Reply:
I copied the title from the blog, sounds like a germanism to me since we say deinstallieren... Removed it now